hierarchy/tString.h

```diff
@@ -0,0 +1,22 @@
+//Tristan Gomez
+//
+//2/7/2020
+
+class tString
```

Comment:
This class is unnecessary and will needlessly complicate production and use massive amounts of memory as we constantly wrap and unwrap strings. We should remove it.